### DISCUSSION TITLE
Feature Request: Create 32-bit ODBC driver

### DISCUSSION THREAD

**Discussion by zan73** [duckdb/duckdb#5910] [Ideas]
I use a Windows SQL IDE that currently only supports 32-bit ODBC. Would it be possible for releases to publish a Windows 32-bit ODBC driver as well?

**Comment by hannes** [comment]
It should be possible but it would severely limit DuckDB's capabilities to e.g. ~4 GB of RAM max. Are you sure this makes sense in your use case?

**Reply by zan73** [reply]
The limitation would possibly cause some issues but I'd like to have the ability at least to use ODBC with my favoured SQL IDE.
If there were some published instructions on how to compile it myself I'd be happy to do. Is there any instructions for compiling for 64-bit ODBC? If so. I could try to figure out how to do it for 32-bit.

**Reply by Mytherin** [reply]
Have a look at the [ODBC CI workflow](https://github.com/duckdb/duckdb/blob/master/.github/workflows/Windows.yml#L153)
